src/pages/InputPage.tsx

```diff
@@ -94,13 +98,19 @@ function InputPage({ onNext }: InputProps) {
 								}
 							/>
 						</div>
-						{error && <p className="error">{error}</p>}
```

Comment:
@JasonWarrenUK I implemented all your changes apart from this line I couldn't see how it fits in with the rest, feel free to make a suggestion that I can commit

Reply:
So this line basically means:

if (there is an error message) {
	show this p element
}

Reply:
So I think you can keep it as is

Reply:
```suggestion
						{error && <p className="error">{error}</p>}
```